templates/toil-instance.yaml

```diff
@@ -185,11 +275,17 @@ Resources:
       UserData:
         Fn::Base64: !Sub |
           #!/bin/bash
+          export DEBIAN_FRONTEND=noninteractive
+          /usr/bin/apt-get -y update
+          /usr/bin/apt-get -y upgrade
+          /usr/bin/apt-get -y dist-upgrade
```

Comment:
if possible i would probably not run  `dist-upgrade` because that is more likely to break things than just running `upgrade`.  more info here: https://askubuntu.com/questions/194651/why-use-apt-get-upgrade-instead-of-apt-get-dist-upgrade